### PR TITLE
Use Cypress GitHub action

### PR DESCRIPTION
**Pull request by bahmutov**
Added example with single and parallel tests using new version of Cypress.io GitHub Action https://github.com/cypress-io/github-action

I say we start using Cypress GitHub Action and then figure out how to show it in our table of CIs (since we also have "normal" GitHub Action examples without our Cypress Action)

Good to go - the failures are flake

### CODE REVIEW

File: .github/workflows/using-action.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Note: make sure to use the same version of cypress-io/github-action
+# in all jobs, otherwise the last version wins I believe
+name: Using Cypress GH Action
+on: [push, pull_request]
```
Comment (kevinold) [inline .github/workflows/using-action.yml]: @bahmutov 

Maybe filter branch on push? https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

```
on:
  push:
    branches:    
      - master         # Push events on master branch
  pull_request:
      - '*'
```

Comment (bahmutov) [reply .github/workflows/using-action.yml]: yeah, this is an excellent idea, especially since we open pull requests every time and don't need push

**Comment by bahmutov** [comment]
yeah, unfortunately the limit for free public projects is pretty low right now, not much we can do, just reinstalls the dependencies :(

**Comment by kevinold** [comment]
@bahmutov Ok, so nothing we can do about it since it's open sourced, so good to merge?

**Comment by bahmutov** [comment]
yes, good to merge - and I will give feedback to GitHub that higher limits would be very nice

**Pull request merged**